### ISSUE TITLE
Butter.js disables sticky header

### ISSUE THREAD

**Issue by udoro** [BCJdevelopment/butter.js#1]
Hello, I love your script and used it in one of my websites. I noticed that my sticky headers are no longer appearing. Do you have a possible solution for this?

**Comment by udoro** [comment]
Never mind.. my bad.. I removed the header from the .butter div and that resolved it.

**Comment by benbcjdev** [comment]
Yeah, I'm glad it worked out for you. All sticky/fixed elements should be positioned outside the container. If you have sticky elements that are supposed to be positioned further down in the container (not at the very top of the viewport like a header might be), then you may have to get creative with JavaScript since sticky elements inside of fixed elements don't stick like they normally would.

I would love to see the website though if you don't mind sharing!

**Comment by udoro** [comment]
thanks for the tip
https://school-masters.net/
